File: 1-even-or-odd/Haskell/EvenOrOdd.hs
module Main where

import System.IO 
import Data.Time.Clock

evenOrOdd :: Integral a => a -> [Char]
evenOrOdd n
  | even n    = "Even"
  | otherwise = "Odd"

main :: IO ()
main = do
  start <- getCurrentTime
  let result = evenOrOdd 1
  putStrLn $ show "res: " ++ show result
  end <- getCurrentTime
  print (diffUTCTime end start)

File: 1-even-or-odd/C/even_or_odd.c
#include<stdio.h>
 
const char* even_or_odd(int number)
{
  return number % 2 == 0 ? "Even" : "Odd";
}

int main() 
{
	const char* even = even_or_odd(12);
	const char* odd = even_or_odd(11);

	printf("Even: %s\n", even);
	printf("Odd: %s\n", odd);

  return 0;
}
